apis/apiextensions/v1alpha1/ccrd/crd.go

```diff
@@ -35,18 +35,12 @@ import (
 	"github.com/crossplane/crossplane/apis/apiextensions/v1alpha1"
 )
 
-// The kind of a published infrastructure resource is the kind of the defined
-// infrastructure resource combined with these suffixes.
 const (
-	PublishedInfrastructureSuffixKind     = "Requirement"
-	PublishedInfrastructureSuffixListKind = "RequirementList"
-	PublishedInfrastructureSuffixSingular = "requirement"
-	PublishedInfrastructureSuffixPlural   = "requirements"
-)
-
-const (
-	errNewSpec         = "cannot generate CustomResourceDefinition from crdSpecTemplate"
-	errParseValidation = "cannot parse validation schema"
+	errNewSpec                 = "cannot generate CustomResourceDefinition from crdSpecTemplate"
+	errParseValidation         = "cannot parse validation schema"
+	errInvalidClaimNames       = "invalid resource claim names"
+	errMissingClaimNames       = "missing names"
```

Comment:
nitpick
```suggestion
	errMissingClaimNames       = "missing resource claim names"
```

Reply:
This is actually by design, because it gets wrapped by InvalidClaimNames and thus reads "invalid resource claim names: missing names".